Doctrine/RouteConditionMetadataListener.php

```diff
@@ -49,6 +49,11 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
         }
 
         $meta = $eventArgs->getClassMetadata();
+
+        if (null === $meta->getReflectionClass()) {
```

Comment:
When initializing ClassMetadata, the reflection is not automatically initialized. for instance, this is used in the DisconnectedClassMetadataFactory used in `doctrine:generate:entities`.

This allows the `doctrine:generate:entities` command to create the PHP code for not yet exisiting entities (e.g. when [reverse-engineering from a non-doctrine database](http://symfony.com/doc/current/cookbook/doctrine/reverse_engineering.html), which is the main use-case for this command).